tests/test_configspace_utils.py

```diff
@@ -46,7 +46,7 @@
       "probabilities": null
     },
     {
-      "name": "lambda",
+      "name": "lmbd",
```

Comment:
It doesn't really matter in this context here, but I often see the pattern of using a underscore-suffix in case of naming collisions with build-ins etc. Like `lambda_`.

Reply:
That's actually also what I had in mind for how to handle these collisions in practice. I'll reflect it in this test already just as an example. Thanks for bringing this up :relaxed: